tests/plugins-tester/src/PluginsTester.js

```diff
@@ -44,29 +41,11 @@ export class PluginsTester {
 	checkPlugins() {
 		const errors = [];
 		this.options.pluginsToTest.forEach( ( slug ) => {
-			try {
-				const filename = process.env.CI ? '../../logs.txt' : 'logs.txt';
-
-				if ( fs.existsSync( filename ) ) {
-					fs.unlinkSync( filename );
-				}
-
-				this.runWP( `npx wp-env run cli 'bash elementor-config/activate_plugin.sh ${ slug } 2>>logs.txt' ` );
```

Comment:
@matipojo why did you delete this code? We spent time to display warnings as annotations.